indico/migrations/versions/20200324_1004_a3295d628e3b_migrate_event_labels_from_settings.py

```diff
@@ -0,0 +1,77 @@
+"""Migrate event labels from settings
+
+Revision ID: a3295d628e3b
+Revises: 266d78b1c5db
+Create Date: 2020-03-24 10:04:10.926079
+"""
+
+import json
+from collections import defaultdict
+from uuid import uuid4
+
+from alembic import context, op
+
+
+# revision identifiers, used by Alembic.
+revision = 'a3295d628e3b'
+down_revision = '266d78b1c5db'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if context.is_offline_mode():
+        raise Exception('This upgrade is only possible in online mode')
+    conn = op.get_bind()
+    # label definitions
+    mapping = {}
+    res = conn.execute("SELECT name, value FROM indico.settings WHERE module = 'event_labels'")
+    for uuid, data in res:
+        assert uuid == data['id']
```

Comment:
Why do you check it here?

Reply:
probably not really needed (since it should never be different) but can't hurt either

Reply:
Yeah, I was actually thinking when these values can be different.